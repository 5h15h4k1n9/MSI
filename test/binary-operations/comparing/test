5 == 5
3 != 4
6 < 10
10 <= 10
8 > 2
5 >= 6
exit
