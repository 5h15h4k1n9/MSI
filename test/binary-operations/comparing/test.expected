(bool) true
(bool) true
(bool) true
(bool) true
(bool) true
(bool) false
Bye-bye!
